### ISSUE TITLE
Themes: Missing screenshot on theme page

### ISSUE THREAD

**Issue by nelsonec87** [Automattic/wp-calypso#59255]
### Details

On the Baxter theme page, `/theme/baxter/<site>` ,the left screenshot is missing

![image](https://user-images.githubusercontent.com/3801502/146254239-ce7248c3-e1f2-4d77-93a2-c0f0fb4581a4.png)

**Comment by sixhours** [comment]
Odd, the theme was missing its Featured Image. I re-set one in the Theme Showcase, but it needs to be resized to better fill out the space:

<img width="1677" alt="Screen Shot 2021-12-15 at 4 44 13 PM" src="https://user-images.githubusercontent.com/2124984/146269648-ca751457-1eff-4872-aac9-e69cb314e6d4.png">

@alaczek @onuro I think this is the same issue we just had in #59152 🤔 The screenshot version I found in the Media Library uploaded on Dec. 14th does not seem to work.

I've added you both to the testing pool in D71467-code so you all can see the themes in the Showcase for yourselves when that's deployed. Sorry to play go-between here. 😅
